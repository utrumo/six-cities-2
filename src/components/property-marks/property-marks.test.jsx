import React from 'react';
import renderer from 'react-test-renderer';
import PropertyMarks from './property-marks.jsx';

it(`PropertyMarks should renders correct`, () => {
  const tree = renderer.create(<PropertyMarks isPremium={true} />).toJSON();
  expect(tree).toMatchSnapshot();
});
