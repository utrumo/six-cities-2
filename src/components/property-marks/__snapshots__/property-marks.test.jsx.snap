// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyMarks should renders correct 1`] = `
<div
  className="property__mark"
>
  <span>
    Premium
  </span>
</div>
`;
